docs/CONTRIBUTING.md

```diff
@@ -0,0 +1,67 @@
+# Contributing guidelines
+
+The rtl_433 project accepts contributions.
+Please check if your contribution is following these guidelines
+to improve the feedback loop and decrease the burden for the maintainers.
+
+## Commit messages
+
+The commit messages should follow the described format.
+
+<area_of_work>: <verb> <commit_message>
+
+Area of work may be one of the following:
+
+- build: for build/build system related work
+- docs: for documentation related work, both in code and readme/docs folder
+- ci: for work related to continuous integration
+- test: for test related work 
+- <decoder_name>: for decoder related work
+- decoders: when multiple decoders are changed at once
+- deps: for changes related to (external) dependencies (e.g. soapysdr is updated or mongoose is updated)
+- core: for changes related to the core decoding and parsing logic (e.g. bitbuffer parsing, fsk decoding)
+- minor: for housekeeping work, code style changes
```

Comment:
I use cosmetics for things that are non functional changes. Indentation for example.

Reply:
Let's leave it at cosmetics then.